website/content/docs/secrets/databases/mysql-maria.mdx

```diff
@@ -217,5 +217,16 @@ GRANT CREATEROLE ON <database>.<object> TO "test-user"@"%";
         service_account_json="@my_credentials.json"
     ```
 
-Once the connection has been configured and IAM authentication is complete, the steps to set up a role and generate
-credentials are the same as the ones listed above.
+1.  Configure a role in Vault as done above. Here you can override Vault's default revocation statements to simply
+    drop the user:
```

Comment:
```suggestion
1. Configure a new role in Vault but override the default revocation statements
   so Vault will drop the user instead:
```
Style correction: write in declarative statements

Reply:
Updated, thanks!